python/dolfinx/fem/function.py

```diff
@@ -420,6 +420,8 @@ def interpolate(
         self,
         u: typing.Union[typing.Callable, Expression, Function],
         cells: typing.Optional[np.ndarray] = None,
+        cell_map: typing.Optional[np.ndarray] = None,
+        expr_mesh: typing.Optional[Mesh] = None,
```

Comment:
`expr_mesh` isn't documented. It would be cleaner if there was a way to avoid passing it

Reply:
I don't see any way to do this, as an expression is not tied to a `Mesh`, after https://github.com/FEniCS/dolfinx/pull/2730

Reply:
Ah OK, in that case `expr_mesh` just needs documenting then